src/js/entities-service/patients.js

```diff
@@ -18,6 +19,16 @@ const Entity = BaseEntity.extend({
 
     return this.fetchModel(id, { data });
   },
+  fetchPatients() {
+    const include = [
+      'groups',
+      'patient-fields',
+    ].join(',');
+
+    const data = { include };
+
+    return this.fetchCollection({ url: '/api/patients', data });
```

Comment:
We shouldn't need to pass the url.  It should be defined on the collection

Reply:
yeah, that's what I thought but it didn't seem to work without it. I get a `Uncaught Error: A "url" property or function must be specified`

Reply:
define the url on the patient entity collection.  we'll only pass the url when it isn't specific to the resource